format/nexus.py

```diff
@@ -81,6 +83,45 @@ class NXValidationError(RuntimeError):
     """A specific exception to record validation errors"""
 
 
+def find_classes(node: NXNode, *nx_classes: Optional[str]) -> Tuple[List[h5py.Group]]:
+    """
+    Find instances of multiple NXclass types within the children of the current node.
+
+    Args:
+        node: The input h5py node (h5py.File or h5py.Group).
+        nx_classes: Names of NXclass types to search for.
+
+    Returns:
+        A list of matching nodes for each of the specified NXclass types.
```

Comment:
It actually returns a tuple of lists

Reply:
Yes, but the return type is specified in the function signature, so I didn't think it necessary to labour the point here, since any API documentation will pick that up automatically.  I felt that, since functions that return multiple things usually return a tuple of those things, it was more important to describe succinctly what each of those things is.  If you prefer, I could reword it to 'A tuple containing a list of matching nodes for each of the specified NX_class types.'